code/modules/vehicles/armored/collision.dm

```diff
@@ -0,0 +1,93 @@
+/**
+  *This proc is called when a atom is crashed into by a [armored vehicle][/obj/vehicle/armored]. Damage is then dealt to both the vehicle and atom
+  *
+  * * Arguments:
+  * * veh is the vehicle that is ramming
+  * * facing is the direction the vehicle is facing for when we ram it
+  * * T is the turf where the vehicle is used with-
+  * * temp to check whether a mob is squished
+  */
+/atom/proc/vehicle_collision(obj/vehicle/veh, facing, turf/T, turf/temp)
+	if(!veh.demolish_on_ram)
+		return NONE // 0 Damage
+
+	var/damage = veh.ram_damage // Each vehicle gets its own damage
+	var/veh_damage = 25
+
+	if(world.time > veh.lastsound + 1 SECONDS)
+		visible_message("<span class='danger'>[veh] crushes the [src]!</span>")
+		playsound(src, 'sound/effects/metal_crash.ogg', 45)
+		veh.lastsound = world.time
+		veh.take_damage(veh_damage)
```

Comment:
Health increases instead of decreasing.

Reply:
It doesn't the indicator is erroneous 
Also please use issues or comments for issues and not reviews